shared/api/server.go

```diff
@@ -217,15 +217,19 @@ type ServerUntrusted struct {
 	//
 	// API extension: client_cert_presence
 	ClientCertificate bool `json:"client_certificate" yaml:"client_certificate"`
+
+	// Server configuration map limited to publicly visible keys (refer to doc/server.md)
+	// Example: {"user.microcloud": ":true"}
+	Config map[string]any `json:"config" yaml:"config"`
 }
 
 // Server represents a LXD server
 //
 // swagger:model
 type Server struct {
 	WithEntitlements `yaml:",inline"`
-	ServerPut        `yaml:",inline"`
 	ServerUntrusted  `yaml:",inline"`
+	ServerPut        `yaml:",inline"`
```

Comment:
Needed to switch the order here, so the doc for the config field from `ServerUntrusted` gets correctly overridden with the one from `ServerPut`